unlock-app/src/components/interface/buttons/Button.js

```diff
@@ -8,63 +8,59 @@ const clickAction = (e, action) => {
   if (action) action()
 }
 
-export const LayoutButton = ({ href, title, children, action, ...props }) => {
-  const button = (
-    <Button href={href} onClick={e => clickAction(e, action)} {...props}>
+export const LayoutButton = ({
+  children,
+  backgroundColor,
+  backgroundHoverColor,
+  fillColor,
+  fillHoverColor,
+  ...props
+}) => {
+  return (
+    <BaseButton
+      backgroundColor={backgroundColor || 'var(--grey)'}
```

Comment:
Do we need this if `defaultProps` is being set below?

Reply:
I think you're right. Let me remove that.